src/main/java/com/google/sps/DataServlet.java

```diff
@@ -66,6 +66,8 @@
   // to the list [0, mutList.size() - 1].
   HashMap<String, List<Integer>> mutationIndicesMap = new HashMap<>();
 
+  HashMap<String, Set<Integer>> tokenIndicesMap = new HashMap<>();
```

Comment:
nit: comment on what this mapping is for

Reply:
think this was taken care of in optimization, my bad!